### PR TITLE
XIVY-14810 Fix ConditionTypes and enable empty selectOption

### PR DESCRIPTION
**Pull request by ivy-edp**
- Updated the ConditionTypes to properly work with EL expressions.
- Added isFalse to ConditionTypes for Boolean
- Enhanced the static options of the Select component to allow adding one empty selection option.

### CODE REVIEW

**Comment by ivy-edp** [comment]
It’s now possible for the radio button to have an empty option as well, though it might not be very useful in that case. Should I add an option to SelectTableField to allow choosing whether to include an empty option or not? I'm concerned that this might overcomplicate the component.

File: packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx [hunk 1]
```diff
@@ -52,7 +65,7 @@ export const TableField = <TData extends object>({ label, data, onChange, column
   };
 
   const showAddButton = () => {
-    if (tableData.filter(obj => deepEqual(obj, emptyDataObject)).length === 0) {
+    if (tableData.filter(obj => deepEqual(obj, emptyDataObject)).length <= 1) {
```
Comment (ivy-lli) [inline packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx:68]: <==

File: packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx [hunk 2]
```diff
@@ -27,7 +27,20 @@ export const TableField = <TData extends object>({ label, data, onChange, column
...
+      const filteredData = change.filter(obj => {
+        if (deepEqual(obj, emptyDataObject)) {
+          if (emptyDataCount === 0) {
+            emptyDataCount++;
+            return true;
+          }
+          return false;
+        }
+        return true;
+      });
```
Comment (ivy-lli) [inline packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx:41]: why work with a counter and not simply by boolean flag?

File: packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx (hunk 1 above)
Comment (ivy-edp) [reply packages/editor/src/editor/sidebar/fields/table-field/TableField.tsx:68]: for some reason this does not work:
grafik

**Pull request merged**